src/graph/query_graph.h

```diff
@@ -58,6 +59,9 @@ QGEdge *QueryGraph_GetEdgeByAlias(const QueryGraph *qg, const char *alias);
 /* Determine whether a given alias refers to a node or relation. */
 EntityType QueryGraph_GetEntityTypeByAlias(const QueryGraph *qg, const char *alias);
 
+/* Validates that query graph's entities label/relationship ids are updated. */
+void QueryGraph_Update(QueryGraph *g);
+
```

Comment:
The name `QueryGraph_Update` is too general, rename to `QueryGraph_ResolveUnknownIDs`
Also I'm not sure if this is the right place for this function, I see this operation as an external procedure acting on a query graph.